Lombiq.Tests.UI/SecurityScanning/SecurityScanningConfiguration.cs

```diff
@@ -32,23 +31,22 @@ public class SecurityScanningConfiguration
     /// </summary>
     public Action<UITestContext, SarifLog> AssertSecurityScanResult { get; set; } = AssertSecurityScanHasNoAlerts;
 
+    /// <summary>
+    /// Gets a value indicating whether the security scan should not visit the <see cref="ErrorController"/> to test
+    /// for correct error handling. This is achieved by running the scan a second time without leaving that page.
+    /// </summary>
+    public bool DontScanErrorPage { get; private set; }
```

Comment:
This would be better in `SecurityScanConfiguration`. That's about configuring a single scan; this class is about configuring all scans and default configs.

Reply:
I thought this setting should apply to all scans. But okay, moved.